### PR TITLE
Update PG_HOST in revert job

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: openshift/deploy/job/schema-revert.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ objects:
           - name: PGPORT
             value: "5432"
           - name: PGHOST
-            value: ${PREFIX}postgres
+            value: cas-postgres-master
```
Comment (wenzowski) [inline openshift/deploy/job/schema-revert.yml:67]: `${PREFIX}postgres-master`, no?

**Review by wenzowski — CHANGES_REQUESTED** [review]

File: openshift/deploy/job/schema-revert.yml (hunk 1 above)
Comment (matthieu-foucault) [reply openshift/deploy/job/schema-revert.yml:67]: ${PREFIX} is 'cas-ggircs' in this repo...

Comment (wenzowski) [reply openshift/deploy/job/schema-revert.yml:67]: gotcha

**Review by wenzowski — APPROVED** [review]
:shipit:

**Pull request merged**